data_override/data_override.F90

```diff
@@ -367,7 +371,14 @@ subroutine read_table(data_table)
 
 !  Read coupler_table
     open(newunit=iunit, file='data_table', action='READ', iostat=io_status)
-    if(io_status/=0) call mpp_error(FATAL, 'data_override_mod: Error in opening file data_table')
+    if(io_status/=0) then
+      if(io_status==29) then
```

Comment:
Is this compiler or system dependent?

Reply:
ugh, you're right, I thought I was being clever here and using a "standard". I guess I can just use INQUIRE instead?

Reply:
I've now pushed a change that should also obviate the need to go through the reading table portion if the file doesn't exist... let me know if anything looks like it won't work with that.